src/BarrelPlanarMPGDTracker_geo.cpp

```diff
@@ -320,4 +320,5 @@ static Ref_t create_MPGDDIRC_geo(Detector& description, xml_h e, SensitiveDetect
 
 //@}
 // clang-format off
-DECLARE_DETELEMENT(epic_MPGDDIRC, create_MPGDDIRC_geo)
+DECLARE_DETELEMENT(epic_OuterMPGDBarrel, create_BarrelPlanarMPGDTracker_geo)
+DECLARE_DETELEMENT(epic_MPGDDIRC, create_BarrelPlanarMPGDTracker_geo)
```

Comment:
Unused now(?).

Reply:
This plug in is used by the new outer MPGD barrel and is now also used by the MPGD DIRC layer which was used in arches. Line 324 was left for comparability with arches config.